src/render_engine/gl3/glad.c

```diff
@@ -1,32 +1,35 @@
 /*
```

Comment:
Why do we have glad in tree at all/why isn't this in external?

Reply:
Because GLAD

GLAD is meant for you to include it in the tree

If you want to wrap it up in a CMake and PR it in that's fine but I have no desire at all to do any of that

Reply:
lmao, ugh, if I care I'll fix it tmmr.